lambdas/services/process_mns_message_service.py

```diff
@@ -20,11 +21,13 @@
 class MNSNotificationService:
     def __init__(self):
         self.document_service = DocumentService()
-        self.table = os.getenv("LLOYD_GEORGE_DYNAMODB_NAME")
+        self.lg_table = os.getenv("LLOYD_GEORGE_DYNAMODB_NAME")
+        self.document_review_table = os.getenv("DOCUMENT_REVIEW_DYNAMODB_NAME")
```

Comment:
Should the MNSNService have responsibility for the lg/document_review table? shouldn't the relevant services isolate these values?